Assets/Localization/StringTables/Internal_RSC_en.asset

```diff
@@ -16649,7 +16649,7 @@ MonoBehaviour:
       if he did consider you a friend, the Emperor''s
       word was[/left]
 
-      law, and you boarded a ship that very night for[/left]
+      law, and you boarded a ship the very next day for[/left]
```

Comment:
Is this change done to keep this consistent with other strings? Because I otherwise don't see why "that very night" would be changed.

Reply:
I guess that's for consistency with the Emperor's speech 

> Now, my champion, rest well tonight, for tomorrow you sail for the kingdom of Daggerfall.

Reply:
Precisely. Although, that intro movie contains a few other lines inconsistent with the story, such as saying that he sent a letter to the Queen of Daggerfall 'several years ago' when the ingame text says it was only about one year ago. He also calls you a friend despite a few of the backstories having you meet the Emperor very shortly before he sends you on this quest.